src/main/java/poomasi/domain/farm/service/FarmFarmerService.java

```diff
@@ -0,0 +1,50 @@
+package poomasi.domain.farm.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import poomasi.domain.farm.dto.FarmRegisterRequest;
+import poomasi.domain.farm.dto.FarmUpdateRequest;
+import poomasi.domain.farm.entity.Farm;
+import poomasi.domain.farm.repository.FarmRepository;
+import poomasi.global.error.BusinessException;
+
+import static poomasi.global.error.BusinessError.*;
+
+@Service
+@RequiredArgsConstructor
+public class FarmFarmerService {
+    private final FarmRepository farmRepository;
+
+    public Long registerFarm(FarmRegisterRequest request) {
+        // TODO: 판매자 인가?
+
+        farmRepository.getFarmByOwnerIdAndDeletedAtIsNull(request.memberId()).ifPresent(farm -> {
+            throw new BusinessException(FARM_ALREADY_EXISTS);
+        });
+
+        return farmRepository.save(request.toEntity()).getId();
+
+    }
+
+    public Long updateFarm(Long farmerId, FarmUpdateRequest request) {
+        Farm farm = this.getFarmByFarmId(request.farmId());
+
+        if (!farm.getOwnerId().equals(farmerId)) {
+            throw new BusinessException(FARM_OWNER_MISMATCH);
+        }
```

Comment:
ownerId 를 get 메서드를 통해 가져와서 비교하지 말고 Farm 객체에 시키면 어떨까요?

```suggestion
        if (farm.isOwner(famerId)) {
            throw new BusinessException(FARM_OWNER_MISMATCH);
        }
```

Reply:
오!! 좋습니다 Farm객체로 옮기도록 하겠습니다~~